python/test_component.py

```diff
@@ -0,0 +1,16 @@
+import sys
+from v9 import V9Component
+
+
+def test(http_method, path, request_arguments, request_body):
```

Comment:
Does this design allow for a developer to edit/populate the response to the request?

Reply:
Methods return a `response_code, response_body` tuple. Is that what you mean?

I will add documentation

Reply:
Oh yes, that makes sense. Adding documentation would be good.